.stylelintrc.json

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": "stylelint-config-recommended",
```

Comment:
Does it one support the "kebab-case"?

Reply:
Done. You have to do it using regex.  

https://stackoverflow.com/questions/58499638/integrate-stylelint-with-angular-build-and-make-it-failed-on-error

Reply:
you would take a look at this one

https://github.com/stylelint/stylelint/blob/master/docs/user-guide/faq.md#how-do-i-configure-the--pattern-rules-for-common-css-naming-conventions-like-kebab-case

Reply:
Yeah, I used the same regex in the link you shared:
```"selector-class-pattern": ["^([a-z][a-z0-9]*)(-[a-z0-9]+)*$", {
      "resolveNestedSelectors": true
    }]```

Reply:
Also help me to add to this file. It's the "github workflow"

https://github.com/davidnguyen179/storybook-wdio/blob/master/.github/workflows/ci.yml

Reply:
Done.